analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -186,21 +186,16 @@ class ClearlyDefinedPackageCurationProvider(
 private fun SourceLocation?.toArtifactOrVcs(): Any? =
     this?.let { sourceLocation ->
         when (sourceLocation.type) {
-            ComponentType.GIT -> {
-                VcsInfoCurationData(
-                    type = VcsType.GIT,
-                    url = sourceLocation.url,
-                    revision = sourceLocation.revision,
-                    path = sourceLocation.path
-                )
-            }
-
-            else -> {
-                // TODO: Implement provider-specific mapping of coordinates to URLs.
-                RemoteArtifact(
-                    url = sourceLocation.url.orEmpty(),
-                    hash = Hash.NONE
-                )
-            }
+            ComponentType.GIT -> VcsInfoCurationData(
+                type = VcsType.GIT,
+                url = sourceLocation.url,
+                revision = sourceLocation.revision,
+                path = sourceLocation.path
+            )
+            // TODO: Implement provider-specific mapping of coordinates to URLs.
```

Comment:
With the previous comment applied, the `TODO` should here land above line 197, and current line 195 should stay empty to separate cases.